packages/storybook/src/schematics/configuration/lib-files/.storybook/config.js__tmpl__

```diff
@@ -0,0 +1,8 @@
+import { configure } from '<%= offsetFromRoot %>../.storybook/config';
+
+function loadStories() {
+  const req = require.context('../src/lib', true, /\.stories\.ts$/);
```

Comment:
`const req = require.context('../src/', true, /\.stories\.ts$/);`
Think this would be better incase someone has an app level storybook

Reply:
Sure, that's a good idea.